lib/rules/jsdoc.js

```diff
@@ -22,8 +22,8 @@
  *
  * There are 3 presets for `Closure Compiler`, `JSDoc3` and `JSDuck5`.
  *
- * By default it allows any tag of mixed set. You can pass `Object`
- * to select preset with `preset` field and add custom tags with `extra` field.
+ * By default it allows any tag of mixed set. You can pass `Object` to select
+ * preset with `preset` field and add custom tags with `extra` field.
```

Comment:
Mixed set is a merged set of all tag files (presets). Union in terms of Set theory.